azure_media_services/static/js/player.js

```diff
@@ -46,7 +46,14 @@ function AzureMediaServicesBlock(runtime, element) {
           var divContainer = $("<div class='azure-media-player-toggle-button-style fa fa-quote-left' id='toggleTranscript' role='button' aria-live='polite' tabindex='0'><div class='vjs-control-content'><span class='vjs-control-text'>Toggle</span></div></div>");
           $(".amp-controlbaricons-right").append(divContainer);
 
-          $('#toggleTranscript').click(function() {
+          $('#toggleTranscript').on('click keydown',(function(evt) {
+		  var keycode = (evt.type === 'keydown' && evt.keycode ? evt.keyCode : evt.which) 
+		  if (evt.type !== 'click' && (keycode !== 32 && keycode !== 13))
+			  return;
+	  }
```

Comment:
Low priority but could you clean up formatting of this code? Use a tab with 4 spaces instead of 8 and always use the same indentation guidelines.

Make sure you adhere to the edx developers guide: 
http://edx.readthedocs.io/projects/edx-developer-guide/en/latest/style_guides/javascript_guidelines.html 
Maybe consider using https://github.com/pangloss/vim-javascript 

Short term check out this solution for improving vim.
http://stackoverflow.com/questions/234564/tab-key-4-spaces-and-auto-indent-after-curly-braces-in-vim